tasks/post-to-cmr/index.js

```diff
@@ -114,16 +115,17 @@ async function postToCMR(event) {
     const token = await launchpad.getLaunchpadToken(event.config.launchpad);
     cmrCreds.token = token;
   } else {
+    const secret = await secretsManager().getSecretValue({
```

Comment:
I mentioned in an earlier ticket, but I'd rather see `getCreds` removed from `cmrjs`. I also left it the way it is because I didn't want to change the interface or behavior of `post-to-cmr` in this PR, I'm just trying to get rid of the bootstrap script.